examples/auth0/.env.local.example

```
@@ -1,12 +1,12 @@
 # Public Environment variables that can be used in the browser.
-NEXT_PUBLIC_AUTH0_CLIENT_ID=
+NEXT_PUBLIC_AUTH0_CLIENT_ID=TEST
 NEXT_PUBLIC_AUTH0_SCOPE="openid profile"
-NEXT_PUBLIC_AUTH0_DOMAIN=
+NEXT_PUBLIC_AUTH0_DOMAIN="http://example.com"
 NEXT_PUBLIC_BASE_URL="http://localhost:3000"
 NEXT_PUBLIC_REDIRECT_URI="/api/callback"
 NEXT_PUBLIC_POST_LOGOUT_REDIRECT_URI="/"
 
 # Secret environment variables only available to Node.js
-AUTH0_CLIENT_SECRET=
-SESSION_COOKIE_SECRET=
+AUTH0_CLIENT_SECRET="this-is-a-secret-value-with-at-least-32-characters"
+SESSION_COOKIE_SECRET="this-is-a-secret-value-with-at-least-32-characters"
```

```suggestion
AUTH0_CLIENT_SECRET=
SESSION_COOKIE_SECRET=
```

Let's not add suggestions to avoid someone forgetting to set these and leave it on the "default" value.